pkg/cmd/factory/default.go

```diff
@@ -91,7 +91,20 @@ func httpClientFunc(f *cmdutil.Factory, appVersion string) func() (*http.Client,
 
 func browser(f *cmdutil.Factory) cmdutil.Browser {
 	io := f.IOStreams
-	return cmdutil.NewBrowser(os.Getenv("BROWSER"), io.Out, io.ErrOut)
+	return cmdutil.NewBrowser(browserLauncher(f), io.Out, io.ErrOut)
+}
+
+// Browser precedence
+// 1. browser from config
+// 2. BROWSER
```

Comment:
Should we have a GH_BROWSER variable that takes top precedence, thus allowing overriding the browser per-call?

Reply:
I think I mentioned wanting to add this in an offline convo and I am 👍 and will add it in.